.github/actions/linux-build/action.yml

```diff
@@ -144,10 +149,6 @@ runs:
       shell: bash
       run: sudo systemctl start mysql.service
 
-    - name: run dbimport
-      shell: bash
-      run: env/dist/bin/dbimport
-
```

Comment:
this should not be necessary imo? doesn't this disable import of pending db files for all ci runs in general